sdk/python/feast/infra/passthrough_provider.py

```diff
@@ -148,10 +148,16 @@ def update_infra(
     ):
         # Call update only if there is an online store
         if self.online_store:
+            tables_to_keep_online = [
+                fv
+                for fv in tables_to_keep
+                if not isinstance(fv, FeatureView) or fv.online
```

Comment:
I'm not sure if everying feature view will have a "online" parameter?

Reply:
I think the API might change in the future, so to avoid breaking, maybe add a check if it has this attribute.